pastastore/store.py

```diff
@@ -300,6 +300,81 @@ def get_nearest_stresses(
             data = pd.concat([data, series], axis=0)
         return data
 
+    def get_signatures(
+        self,
+        signatures=None,
+        names=None,
+        libname="oseries",
+        progressbar=False,
+        ignore_errors=False,
+    ):
+        """Get groundwater signatures. NaN-values are returned when the
+        signature could not be computed.
+
+        Parameters
+        ----------
+        signatures : list of str, optional
+            list of groundwater signatures to compute, if None all groundwater
+            signatures in ps.stats.signatures.__all__ are used, by default None
+        names : str, list of str, or None, optional
+            names of the time series, by default None which
+            uses all the time series in the library
+        libname : str
+            name of the library containing the time series
+            ('oseries' or 'stresses'), by default "oseries"
+        progressbar : bool, optional
+            show progressbar, by default False
+        ignore_errors : bool, optional
+            ignore errors when True, i.e. when non-existent timeseries is
+            encountered in names, by default False
+
+        Returns
+        -------
+        sign : pandas.DataFrame
```

Comment:
sign as a name is a bit confusing, but thats just a minor comment. Most people won't see or be bothered by that.